wiki/Gameplay/Game_modifier/Bloom/en.md

```diff
@@ -0,0 +1,40 @@
+---
+tags:
+  - BM
+---
+
+# Bloom (mod)
+
+::: Infobox
+
+<!-- lint ignore heading-increment -->
+
+#### Bloom
+
+![Bloom mod icon](/wiki/Gameplay/Game_modifier_(lazer)/img/mods/BM.png)
+
+*The cursor blooms into.. a larger cursor!*
+
+|  |  |
+| :-- | :-- |
+| Acronym | BM |
+| Type | Fun |
+| Game modes | ![][osu!] |
+| Score multiplier | 1.00x |
+| Status | Unranked |
+| Incompatible mods | [Flashlight (FL)](/wiki/Gameplay/Game_modifier/Flashlight_(lazer)), [No Scope (NS)](/wiki/Gameplay/Game_modifier/No_Scope), [Touch Device (TD)](/wiki/Gameplay/Game_modifier/Touch_Device_(lazer)) |
+
+:::
+
+*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*
+
+The **Bloom** mod progressively makes the cursor bigger as the combo builds up until it reaches the limit of 50x combo. If you miss an object, the cursor returns to its original size.
```

Comment:
shoudl probably word this in a way where it doesn't depend on mentioning the exact number "50x", bc that is configurable